tests/ProxyInterfaceSourceGeneratorTests/InheritedInterfaceTests.cs

```diff
@@ -0,0 +1,168 @@
+using CSharp.SourceGenerators.Extensions;
+using CSharp.SourceGenerators.Extensions.Models;
+using FluentAssertions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using ProxyInterfaceSourceGenerator;
+using ProxyInterfaceSourceGeneratorTests.Source.Disposable;
+using Xunit.Abstractions;
+
+namespace ProxyInterfaceSourceGeneratorTests
+{
+    public class InheritedInterfaceTests
+    {
+        private const string Namespace = "ProxyInterfaceSourceGeneratorTests.Source.Disposable";
+        private const string OutputPath = "../../../Destination/Disposable/";
+        private readonly ProxyInterfaceCodeGenerator _Dut;
```

Comment:
can you change it to `ProxyInterfaceCodeGenerator _sut;`?
s u t 
system under test